README.md

```diff
@@ -10,19 +10,18 @@ This codeception module is based on [oqq/codeception-email-mailhog](https://gith
 brings nearly the same functionality for Mailpit as the mentioned modules did for MailHog.
 
 ## Installation
-Through composer, require the package:
+With Composer:
 ```shell
-composer req koehnlein/codeception-email-mailpit --dev
+composer require koehnlein/codeception-email-mailpit --dev
 ```
-Then turn it on in your Codeception suite yaml file
+Then enable it in your `Functional.suire.yml`:
 ```yaml
-class_name: FunctionalTester
 modules:
     enabled:
         - Mailpit
     config:
         Mailpit:
-            url: 'http://mailpit.dev'
+            url: 'http://localhost'
```

Comment:
It know that http://mailpit.dev is not a good example, because `.dev` is an existing TLD. But on the other hand, http://localhost is not really much better. I would prefer to have an example where everyone understands that it musst be changed to the real needs of the current setup, something like `https://your-mailpit-instance` or `http://example.org`

Reply:
`http://localhost` is the default URL that Mailpit runs on. So this isn't meant as some placeholder-domain, but rather the actual URL that works for new users.
Here's what I have in my `Functional.suite.yml`:
```yaml
    config:
        Mailpit:
            url: 'http://localhost'
            port: '8025'
```